synapse/models/risk.py

```diff
@@ -13,7 +13,7 @@ def getModelDefs(self):
                     'doc': 'An instance of a vulnerability present in a target.',
                 }),
                 ('risk:threat', ('guid', {}), {
-                    'doc': 'A threat cluster or subgraph of threat activity.',
+                    'doc': 'A threat cluster or subgraph of threat activity, as described by a specific reporter.',
```

Comment:
"reporter" might imply a journalist. maybe change to "as reported by a specific organization" ?